src/main/java/uk/gov/ons/ssdc/caseprocessor/schedule/ClusterLeaderManager.java

```diff
@@ -56,7 +56,7 @@ public boolean isThisHostClusterLeader() {
     ClusterLeader clusterLeader = lockedClusterLeaderOpt.get();
 
     if (clusterLeader.getHostName().equals(hostName)) {
-      log.with("hostName", hostName).info("This host is leader");
+      //      log.with("hostName", hostName).info("This host is leader");
```

Comment:
Code left commented out. Perhaps some of these should be made debug level to reduce noise if they needed to be commented out to work on it?